languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,32 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLE_OU_QUELLES_GROUPE_NOMINAL" name="quelle ou quelles groupe nominal" default="temp_off">
+      <rule>
+          <pattern>
+              <token postag="V.*" postag_regexp="yes"/>
```

Comment:
I'm wondering: is this verb form needed? Shouldn't we catch any adjective or noun after "qu'elle" (unless it's the same as a verb form, e.g. "porte")?

Reply:
...this would make the rules about "qu'elles?" after SENT_START obsolete.